tools/export/cces/__init__.py

```diff
@@ -53,9 +53,12 @@ def is_target_supported(cls, target_name):
         target_name - the name of the target.
         """
         target = TARGET_MAP[target_name]
-        return (cls.TOOLCHAIN in target.supported_toolchains) \
-            and hasattr(target, "device_name") \
-            and (target.device_name in SUPPORTED_DEVICES)
+        if not target.is_TFM_target:
```

Comment:
Where are we documenting that exporters don't work with TF-M targets? Do we ever want exporters to work with TF-M targets in the future? If so, what would it take to make it work (maybe CMake)?

Reply:
Updated documentation to indicate that, Mbed OS exporters are currently not supported for PSA dual-core and Armv8-M targets.

 `Do we ever want exporters to work with TF-M targets in the future?`
This is a product question. If the answer is yes, I think the following could work:

* Use Mbed OS exporters to export Mbed OS related files
* Export TF-M secure side binary
* If we make assembler directives to work then linker will take care of linking both secure side and non-secure side binaries.